### PR TITLE
New package: Gitbutler.GitButler version 0.11.0

### PR DESCRIPTION
**Pull request by IndyV**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

---

### CODE REVIEW

File: manifests/g/Gitbutler/GitButler/0.11.0/Gitbutler.GitButler.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Gitbutler.GitButler
+PackageVersion: 0.11.0
+InstallerLocale: en-US
+InstallerType: wix
+ProductCode: '{7B797A7C-C01B-4219-9190-5905FA70DB80}'
+Installers:
+- InstallerUrl: https://app.gitbutler.com/downloads/release/windows/x86_64/msi
```
Comment (Trenly) [inline manifests/g/Gitbutler/GitButler/0.11.0/Gitbutler.GitButler.installer.yaml:10]:
```suggestion
- InstallerUrl: https://releases.gitbutler.com/releases/release/0.11.0-810/windows/x86_64/GitButler_0.11.0_x64_en-US.msi
```
Versioned URLs are preferred

**Review by Trenly — CHANGES_REQUESTED** [review]
Any additional metadata you can add like the PackageURL would be extremely helpful

**Comment by IndyV** [comment]
Added as much metadata as I could find @Trenly.
Please let me know if there are more fields I should add :)

**Comment by stephengillie** [comment]
Verify: 
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**